src-ts/utils/account/Account.tsx

```diff
@@ -4,7 +4,8 @@ import Modal from 'react-responsive-modal'
 import { authUrlLogin, Button, Card, formOnReset } from '../../lib'
 import { profileContext, ProfileContextData } from '../../lib/profile-provider'
 
-import styles from './Account.module.scss'
+import styles from './AccountSettings.module.scss'
```

Comment:
this import is not grouped correctly. this would fail the lint check. 

if you set your VS code preferences up per these instructions, these issues will be fixed any time you save a file:

https://topcoder.atlassian.net/wiki/spaces/PROD/pages/2811199511/NextGen+MFE+Setup#VS-Code